packages/aws-cdk-lib/aws-stepfunctions/lib/states/fail.ts

```diff
@@ -9,6 +9,8 @@ import { INextable } from '../types';
 export interface FailProps {
   /**
    * Optional name for this state
+   *
+   * @default LogicalResourceId
```

Comment:
I don't think that this is accurate. It's more accurate to say something like:

```ts
* @default - The construct ID will be used as state name
```

Reply:
Okay, will update that. Thanks Rico.